bokehjs/test/unit/core/util/templating.ts

```diff
@@ -173,7 +173,7 @@ describe("templating module", () => {
       expect(s3).to.be.equal("stuff a")
 
       const s4 = tmpl.replace_placeholders("stuff @bar", source, 1)
-      expect(s4).to.be.equal("stuff &lt;div&gt;b&lt;/div&gt;")
+      expect(s4).to.be.equal("stuff <div>b</div>")
```

Comment:
Can we add another row in the CDS with an apostrophe/single quote to make sure the exact case in the issue is exercised?

Reply:
Added.